src/libs/ReportUtils.ts

```diff
@@ -2611,11 +2611,13 @@ function getReportName(report: OnyxEntry<Report>, policy: OnyxEntry<Policy> = nu
         return formattedName;
     }
 
-    // Not a room or PolicyExpenseChat, generate title from participants
-    const participantAccountIDs = report?.participantAccountIDs ?? [];
+    // Not a room or PolicyExpenseChat, generate title from first 5 other participants
+    const participantAccountIDs = report?.participantAccountIDs?.slice(0, 6) ?? [];
```

Comment:
Are we sure the current user is going to be returned in the first six results?

Reply:
I have the same thoughts, but looking and the next line, we filter out the current user `accountID` and pop the rest if it's still six accountIDs.

Reply:
Fair point. I was only looking at the changed code. Still wouldn't it be cleaner/easier if we took a slice of `paricipantsWithoutCurrentUser` to make this?

Reply:
Sorry guys for this delay, just to clarify: this PR was only intended to improve the performance - on big accounts the `.filter()` loop simply took ages and looping through all these items is redundant from the product perspective (please see the pinned proposal and investigation). Hope this helps :)